backend/inspirehep/mailing/providers/mailtrain.py

```diff
@@ -40,3 +42,29 @@ def mailtrain_subscribe_user_to_list(list_id, email, first_name, last_name):
             "User successfully added to the mailing list.", list_id=list_id, email=email
         )
     return response
+
+
+def mailtrain_update_weekly_campaign_content(html_content):
+    """Sends html_content of the campaign to redis WEEKLY_JOBS_EMAIL_REDIS_KEY.
+    Args:
+        html_content (str): HTML content of the campaign.
+    """
+    mailtrain_key = current_app.config.get("WEEKLY_JOBS_EMAIL_REDIS_KEY")
+    if not mailtrain_key:
+        LOGGER.error(
+            "`WEEKLY_JOBS_EMAIL_REDIS_KEY` is not set. MAILTRAIN campaign is not properly configured!"
+        )
```

Comment:
Do we really want to blow up a CLI command with exception? It will still stop processing, but having stacktrace in CLI is awful. Isn't it better that we post error message to sentry and still app exits with nice error and without stacktrace?

Reply:
You are right. Raising exception is better here as we do not store errors in sentry.

Reply:
We probably do, but blowing up allows you to exit with a non-zero code automatically.

Different question: how can this ever not be set, given you're setting it in the config?

Reply:
It can be overwritten by empty string in ENV (some mistake on kubernetes configs maybe?)

Reply:
So you have 6 lines to protect against something that should never happen. I think doing
```
mailtrain_key = current_app.config["WEEKLY_JOBS_EMAIL_REDIS_KEY"]
```
is good enough and simpler.